src/main/java/gregtech/common/metatileentities/multi/electric/MetaTileEntityAssemblyLine.java

```diff
@@ -45,6 +53,7 @@ protected BlockPattern createStructurePattern() {
                 .where('Y', states(getCasingState()).or(abilities(MultiblockAbility.INPUT_ENERGY).setMinGlobalLimited(1)))
                 .where('I', metaTileEntities(MetaTileEntities.ITEM_IMPORT_BUS[0]))
                 .where('G', states(MetaBlocks.MULTIBLOCK_CASING.getState(BlockMultiblockCasing.MultiblockCasingType.GRATE_CASING)))
+                .where('D',abilities(MultiblockAbility.RESEARCH_DATA).or(states(MetaBlocks.MULTIBLOCK_CASING.getState(BlockMultiblockCasing.MultiblockCasingType.GRATE_CASING))))
```

Comment:
Allow the hatch to be on any grate via this.
```suggestion
                .where('G', states(MetaBlocks.MULTIBLOCK_CASING.getState(BlockMultiblockCasing.MultiblockCasingType.GRATE_CASING)).or(abilities(MultiblockAbility.RESEARCH_DATA).setMinGlobalLimited(1).setMaxGlobalLimited(9)))
```